bucket.go

```diff
@@ -288,18 +288,23 @@ func (b *Bucket) Put(key []byte, value []byte) error {
 		return ErrValueTooLarge
 	}
 
+	// Insert into node.
```

Comment:
It's related to https://github.com/etcd-io/bbolt/pull/550.

If it's not accepted, I can use `key` here. Or I can cherry-pick 550 first.

Reply:
Please backport both https://github.com/etcd-io/bbolt/pull/550 and https://github.com/etcd-io/bbolt/pull/637. They are highly related, so let's just backport both of them in this PR.

Reply:
Updated